src/controllers/ModelsController.ts

```diff
@@ -62,6 +62,25 @@ const makeModelsController = (
     });
   });
 
+  ModelsController.getAsync('/advancedSearch', async (req, res) => {
+    //A custom route for complex search since we can't use queryToMongo
```

Comment:
better:    ` //A custom route for complex search that can't be processed by queryToMongo`